src/World3/population/pop15/plots.jl

```diff
@@ -1,6 +1,7 @@
+using DifferentialEquations
 function historicalrunsolution()
     isdefined(@__MODULE__, :_solution_historicalrun) && return _solution_historicalrun
-    global _solution_historicalrun = solve(historicalrun(), (1900, 2100))
+    global _solution_historicalrun = WorldDynamics.solve(historicalrun(), (1900, 2100); solver =  Rodas5())
```

Comment:
This PR removes the `AutoVern9` method on top of `Rodas5`. Keeping it raises a mass matrix issue, but removing it results in oscillating plots. We're keeping `AutoVern9` in all other plots. We keep investigating the issue in #60.